dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -290,6 +292,21 @@ class BGPPeerMgr(Manager):
         vrf, nbr = key.split('|', 1)
         if key not in self.peers:
             cmd = None
+
+            if "local_addr" not in data:
+                syslog.syslog(syslog.LOG_WARNING, 'Peer {}. Error in missing required attribute "local_addr"'.format(key))
+            else:
+                # The bgp session that belongs to a vnet cannot be advertised as the default BGP session.
+                # So we need to check whether this bgp session belongs to a vnet.
+                interface = InterfaceMgr.get_local_interface(self.directory, data["local_addr"])
+                if not interface:
+                    return False
+                vnet = InterfaceMgr.get_vnet(interface)
+                if vnet:
+                    # Ignore the bgp session that is in a vnet
+                    syslog.syslog(syslog.LOG_INFO, 'Peer {} in vnet {}'.format(key, vnet))
```

Comment:
the log message is not proper here. Should be at least a sentence. 
```
"Ignore the BGP peer {} as the interface {} is in vnet {}.format(key, interface, vnet)
```

Reply:
Done it.